R/classes.R

```diff
@@ -47,12 +46,22 @@ print.PartInv <- function(x, ...) {
              colnames(x$summary)[1], "'):\n"))
   print(as.data.frame(lapply(x$ai_ratio, round, digits = 3), row.names = ""))
   cat("\n")
-  summary_print(x$summary)
+  if(dim(x$summary)[2] > 8) {
+    cat("Classification Accuracy Indices:\n")
+    summary_print(x$summary[, 1:(round(dim(x$summary)[2] / 2) + 1)])
```

Comment:
Suggest to compute something like `nc <- ncol(x$summary)` first. You can also use the `ceiling` function to round number up.

Reply:
Thank you for the suggestions, I updated the code accordingly.

Reply:
Should it be `(ceiling(nc / 2) + 1)` (instead of `+ 2`) in the second one?